records/generator_perf_test.go

```diff
@@ -0,0 +1,37 @@
+package records
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+)
+
+func BenchmarkInsertRR(b *testing.B) {
```

Comment:
Does it make sense to add a comment somewhere indicating that this isn't doing taskRecords.